article.py

```diff
@@ -46,7 +46,7 @@ def save_raw(self):
                       indent=4,
```

Comment:
discard changes in the article

Reply:
but the original article.py doesn`t pass lint checks